x11-misc/sunflower/sunflower-0.3_alpha61.ebuild

```diff
@@ -0,0 +1,90 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+inherit gnome2-utils python-r1 xdg-utils
+
+MY_PN="Sunflower"
+DESCRIPTION="Small and highly customizable twin-panel file manager with plugin-support"
+HOMEPAGE="https://github.com/MeanEYE/Sunflower
+	http://sunflower-fm.org/"
+SRC_URI="http://sunflower-fm.org/pub/sunflower-${PV/_alpha/-}.tgz"
+
+LICENSE="GPL-3"
```

Comment:
https://github.com/MeanEYE/Sunflower/blob/develop/application/sunflower.py

Looks GPL-3+ to me.